client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+{
+    "common": {
+        "ok": "OK",
+        "submit": "Übernehmen",
+        "cancel": "Abbrechen",
+        "overwrite": "Überschreiben",
+        "continue": "Weiter",
+        "dismiss": "Verwerfen",
+        "prompt": "Hinweis",
+        "warning": "Warnung",
+        "attention": "Achtung!",
+        "confirm": "Bestätigen",
+        "close": "Schließen",
+        "save": "Speichern",
+        "cancelled": "Abgebrochen",
+        "enabled": "Aktiviert",
+        "commandOK": "Befehl erfolgreich übertragen!",
+        "commandFailed": "Es ist ein Fehler aufgetreten:",
+        "connectionError": "Verbindungsfehler",
+        "m2": "m²",
+        "hours": "Stunde",
+        "hours_plural": "Stunden",
+        "months": ["Jan","Feb","Mär","Apr","Mai","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
+        "days": ["Montag","Dienstag","Mittwoch","Donnerstag","Freitag","Samstag","Sonntag"]
+    },
+    "navi": {
+        "home": "Home",
+        "map": "Karte",
+        "zones": "Zonen",
+        "control": "Steuerung",
+        "settings": "Einstellungen",
+        "links": "Links",
+        "about": "Über"
+    },
+    "home": {
+        "labels": {
+            "area": "Bereich",
+            "time": "Zeit",
+            "battery": "Batterie",
```

Comment:
Wouldn't "Akku" be the correct description?

Reply:
"Akku" is basically the description for "wiederaufladbahre Batterie" (rechargeable battery) I would rather stick to the top description "Batterie", even though it is also correct.

Reply:
Yeah, but as it is a "wiederaufladbare Batterie", "Akku" would be the definition of it ;). However I'm also fine with "Batterie".

Reply:
If I am not mistaken about the context of this one, "Akkustand" might be better?

Reply:
OK, Akku might be better, due to the fact that even my cellphone talks about accu and not battery. -.-

Reply:
I'm struggled over the word "Ladungszustand" IT should be "Ladezustand"